README.md

```diff
@@ -86,11 +86,21 @@ You are welcome and encouraged to run the API locally on your machine. To do so,
 
 ### Dependencies
 
-* SBT (Scala Build Tool) &ndash; available via Homebrew
+* SBT 0.13.x (Scala Build Tool) &ndash; available via Homebrew
```

Comment:
This is truly the smallest nit, and definitely feel free to ignore it, but an [en dash is for numeric ranges](https://typographyforlawyers.com/hyphens-and-dashes.html). We probably want an em dash here (`&mdash;`).

Reply:
Good eye, I'll fix this!